flake.nix

```
@@ -147,13 +173,254 @@
               rm -f ./telegraf/metrics/*.influx 2>/dev/null || true
             '';
           };
+          check-scripts = pkgs.writeShellApplication {
+            name = "check-scripts";
+            runtimeInputs = [
+              pkgs.shellcheck
+            ];
+            text = ''
+              set -euo pipefail
+
+              # shellcheck disable=SC1091
```

Based on https://www.shellcheck.net/wiki/SC1091, could this not be:

```suggestion
              # shellcheck source=scripts/checks.sh
```

Same with the others like this below.